src/AddToCalendar.tsx

```diff
@@ -116,21 +125,21 @@ const AddToCalendar: React.FC<AddToCalendarProps> = ({
   children = "Add to My Calendar",
   event,
   filename = "download",
-  open: initialOpen = false
+  open: initialOpen = false,
 }) => {
   const [open, onToggle] = useOpenState(initialOpen);
   const urls = useMemo<CalendarURLs>(() => makeUrls(event), [event]);
 
   return (
     <div className="chq-atc">
       {event && (
-        <button type="button" className="chq-atc--button" onClick={onToggle}>
-          <svg width="20px" height="20px" viewBox="0 0 1024 1024">
-            <path d="M704 192v-64h-32v64h-320v-64h-32v64h-192v704h768v-704h-192z M864 864h-704v-480h704v480z M864 352h-704v-128h160v64h32v-64h320v64h32v-64h160v128z" />
-          </svg>
-          {" "}
+        <Button
+          className="chq-atc--button"
```

Comment:
Just not sure if you need this class now.. double check, I believe this is could be responsible for the additional button active state